translations/expansion_content.tex/cs.po

```diff
@@ -225,6 +255,25 @@ msgid ""
 "    \\end{itemize}\n"
 "  "
 msgstr ""
+"      \\draw (0, 0) node[inner sep=0] {\\makebox[\\textwidth][c]{\\includegraphics[width=0.8\\textwidth]{\\cards/war_machine.png}}};\n"
+"      \\draw (0.8, 3.6) node {\\encircle{\\phantom{.}1\\phantom{.}}};\n"
+"      \\draw (0, -2.8) node {\\encircle{\\phantom{.}2\\phantom{.}}};\n"
+"      \\draw (-1.2, -1.7) node {\\encircle{\\phantom{.}3\\phantom{.}}};\n"
+"      \\draw (0.9, -1.7) node {\\encircle{\\phantom{.}4\\phantom{.}}};\n"
+"    \\end{tikzpicture}\n"
+"  \\end{scriptsize}\n"
+"  \\break\n"
+"  \\footnotesize{\\textbf{\\textit{\\textcolor{darkcandyapplered}{Karta Válečného Stroje}}}}\n"
+"  \\scriptsize\n"
+"  \\begin{multicols}{2}\n"
+"    \\begin{itemize}\n"
+"      \\item[\\textbf{1.}] \\textbf{Název}\n"
+"      \\item[\\textbf{2.}] \\textbf{Efekt}\n"
+"      \\item[\\textbf{\\phantom{.}}] \\phantom{.}\n"
+"      \\item[\\textbf{3.}] \\textbf{Cena v~Továrně na Válečné Stroje}\n"
```

Comment:
Wow, is it really that long? Try to avoid hyphenation here, because it looks ugly.

Reply:
What about this?
![cs-4](https://github.com/Heegu-sama/Homm3BG/assets/1928291/d7de1d68-4e61-4b9e-b924-0f1d195b33ec)

Reply:
Much better